core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -2081,13 +2083,18 @@ call_wasm_with_hw_bound_check(WASMModuleInstance *module_inst,
         ret = false;
     }
 
+    /* Note: can't check wasm_get_exception(module_inst) here, there may be
+     * exception which is not caught by hardware (e.g. uninitialized elements),
+     * then the stack-frame is already freed inside wasm_interp_call_wasm */
     if (wasm_get_exception(module_inst)) {
```

Comment:
Should be `if (!ret)`

Reply:
forgot to update the code 😂
updated, thanks~